app/components/Article/index.tsx

```diff
@@ -8,17 +8,33 @@ import {Action, ActionType} from '~/routes/questions.actions'
 import type {Glossary, Question} from '~/server-utils/stampy'
 import Contents from './Contents'
 import './article.css'
+import FeedbackForm from '~/components/Article/FeedbackForm'
 
 const isLoading = ({text}: Question) => !text || text === 'Loading...'
 
 const ArticleFooter = (question: Question) => {
+  const [showFeedback, setShowFeedback] = useState(false)
+  const [showFeedbackForm, setShowFeedbackForm] = useState(false)
+  const [isFormFocused, setIsFormFocused] = useState(false)
   const date =
     question.updatedAt &&
     new Date(question.updatedAt).toLocaleDateString('en-GB', {
       year: 'numeric',
       month: 'short',
     })
 
+  React.useEffect(() => {
```

Comment:
why not move all this stuff into the feedback form component? It's only hidden, so the HTML elements will still be around

Reply:
for storybook functionality

Reply:
that's the wrong way round. Storybook, tests etc. shouldn't change how the actual code is structured. If something is hard to do without changes, then it's a hint that something is wrong, but the code should dictate storybook, not the other way round

Reply:
You can add extra wrappers with decorators - check the ArticlesKeepGoing story

Reply:
I've been looking into this but I don't believe it's a feasible option. The article makes it display  when submitted, moving part of the logic to the Feedback component would just duplicate logic and code.